pkg/doc/jsonld/default_contexts.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jsonld
+
+import "embed"
+
+// embedFS contains JSON-LD context documents from the "contexts" folder.
+//go:embed contexts/*.jsonld
+var embedFS embed.FS //nolint:gochecknoglobals
+
+// DefaultContexts contains a list of context documents embedded into Go binary. They can be preloaded into
+// the underlying storage using jsonld.WithContexts() option upon loader creation.
+// TODO: Update with more default contexts.
+var DefaultContexts = []ContextDocument{ //nolint:gochecknoglobals
+	{
+		URL:         "https://w3id.org/security/v1",
```

Comment:
Note: this is the important URL - this is the one that shows up in the documents.